src/game.zig

```diff
@@ -665,6 +685,20 @@ pub fn update(deltaTime: f64) void { // MARK: update()
 			Player.selectedSlot = @intCast(@mod(newSlot, 12));
 			main.Window.scrollOffset = 0;
 		}
+		
+		// View Bobbing
+		// Smooth lerping of bobTime with framerate independent damping
+		const fac: f64 = 1 - std.math.exp(-15 * deltaTime);
+		var targetBobVel: f64 = 0;
+		const horizontalMovementSpeed = vec.length(vec.xy(Player.getVelBlocking()));
+		if (horizontalMovementSpeed > 0.01 and Player.onGround) {
+			targetBobVel = std.math.pow(f64, horizontalMovementSpeed / 4, 0.5);
+		}
+		Player.bobVel = Player.bobVel * (1 - fac) + targetBobVel * fac;
+		if (Player.onGround) { // No view bobbing in the air
+			Player.bobTime += std.math.pow(f64, Player.bobVel, 0.9) * 8 * deltaTime;
```

Comment:
What is the motivation/purpose of the `pow(vel, 0.9)` is?
Could the same be achieved without a power function?

Reply:
This was done so that the speed of the view bobbing would fall off slightly faster than the magnitude, as otherwise it felt quite a lot too fast at high speeds. This could maybe be done with a piecewise linear function, possibly combined with a hard cap.

Reply:
Nvm, after trying without it's not noticably different unless you turn movement speed up to crazy high levels.